deploy/runtime/ui/Battery Lifetime.txt

```diff
@@ -686,6 +686,47 @@ Cycle Degradation
 Bold
 2
 1
+RadioChoice
+
+1
+11
+Name
+5
+15
+batt_life_model
+X
+3
+255
+Y
+3
+84
+Width
+3
+492
+Height
+3
+24
+Tool Tip
+5
+0
+Selection
+3
+0
+Items
+6
+3
+Calendar - Cycle Table
+NMC/Gr Life Model
```

Comment:
If the NMC model is going to be the default, should there be a warning when the user changes away from that chemistry/voltage model?

Reply:
Maybe some kind of messaging in the upgrade script as well?

Reply:
When they change to any chemistry besides NMC and LMO/LTO, it goes back to Calendar - Cycling. Voltage model changes don't do anything